frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts

```diff
@@ -130,8 +130,21 @@ export const gitPage = {
       .get(gitPO.cancel)
       .should('be.enabled')
       .click(),
-  verifyValidatedMessage: () =>
-    cy.get(gitPO.gitSection.validatedMessage).should('have.text', messages.gitUrlValidated),
+  verifyValidatedMessage: (builderImageName = 'Node.js') => {
+    cy.get(gitPO.gitSection.validatedMessage).should('not.have.text', 'Validating...');
+    cy.get('body').then(($body) => {
+      if (
+        $body
+          .find(gitPO.gitSection.validatedMessage)
+          .text()
+          .includes(messages.privateGitRepoMessage)
+      ) {
+        cy.get(`[aria-label="${builderImageName}"]`).click();
```

Comment:
how about logging `selecting builder image builderImageName` in case of fail in validation?

Reply:
Added